test/saithrift/tests/sail3.py

```diff
@@ -184,6 +187,77 @@ def runTest(self):
 
             self.client.sai_thrift_remove_virtual_router(vr_id)
 
+def ip6_to_integer(ip6):
+        ip6 = socket.inet_pton(socket.AF_INET6, ip6)
+        a, b = unpack(">QQ", ip6)
+        return (a << 64) | b
+
+def integer_to_ip6(ip6int):
+        a = (ip6int >> 64) & ((1 << 64) - 1)
+        b = ip6int & ((1 << 64) - 1)
+        return socket.inet_ntop(socket.AF_INET6, pack(">QQ", a, b))
+        
+@group('l3')
+class L3IPv6PrefixTest(sai_base_test.ThriftInterfaceDataPlane):
```

Comment:
can you add a description to describe this test, what does it test? something like test packet forwarding for all IPv6 prefix lenghs (from 0 to 128)